designs/enum-shapes.md

```diff
@@ -49,60 +49,31 @@ enum Suit {
 Each value listed in the enum is a member that implicitly targets
 `smithy.api#Unit`. The string representation of an enum member defaults to the
 member name. The string representation can be customized by applying the
-`@enumValue` trait.
+`@enumValue` trait. We will introduce syntactic sugar to assign the
+`@enumValue` trait to enums and intEnums.
 
 ```
 enum Suit {
-    @enumValue("diamond")
-    DIAMOND
-
-    @enumValue("club")
-    CLUB
-    
-    @enumValue("heart")
-    HEART
-    
-    @enumValue("spade")
-    SPADE
-}
-```
-
-Enums do not support aliasing; all values MUST be unique.
-
-#### enum default values
-
-The default value of the enum shape is an empty string "", regardless of if
-the enum defines a member with a value of "".
-
-```
-structure GetFooInput {
-    @default
-    suit: Suit
+    DIAMOND = "diamond"
+    CLUB = "club"
+    HEART = "heart"
+    SPADE = "spade"
```

Comment:
Did you mean to omit the values here? The shape below that this being compared to is identical

Reply:
Whoops. Meant to add the `@enumValue` trait.